test/specs/runApexTests.e2e.ts

```diff
@@ -0,0 +1,521 @@
+  //   if (successNotificationWasFound !== true) {
+  //     const failureNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Run Apex Tests failed to run');
+  //     if (failureNotificationWasFound === true) {
+  //       expect(successNotificationWasFound).toBe(false);
+  //     }
+  //   } else {
+  //     expect(successNotificationWasFound).toBe(true);
+  //   }
+
+  //   // Verify test results are listed on vscode's Output section
+  //   const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', '=== Test Results', 10);
+  //   expect(outputPanelText).not.toBeUndefined();
+  //   expect(outputPanelText).toContain('=== Test Summary');
+  //   expect(outputPanelText).toContain('TEST NAME');
+  //   expect(outputPanelText).toContain('ExampleApexClass2Test.validateSayHello');
+  //   expect(outputPanelText).toContain('ended SFDX: Run Apex Tests');
+  // });
+
+  step('Run Tests via Command Palette', async () => {
+    const workbench = await browser.getWorkbench();
+
+    // Run SFDX: Run Apex tests.
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Run Apex Tests', 1);
+
+    // Select the "ExampleApexClass1Test" file
+    prompt.selectQuickPick('ExampleApexClass1Test');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests', 5 * 60);
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Listening for streaming state changes...', 5 * 60);
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Processing test run', 5 * 60, false);
+
+    // Look for the success notification that appears which says, "SFDX: Run Apex Tests successfully ran".
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Run Apex Tests successfully ran');
+    if (successNotificationWasFound !== true) {
+      const failureNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Run Apex Tests failed to run');
+      if (failureNotificationWasFound === true) {
+        expect(successNotificationWasFound).toBe(false);
+      }
+    } else {
+      expect(successNotificationWasFound).toBe(true);
+    }
+
+    // Verify test results are listed on vscode's Output section
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', '=== Test Results', 10);
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('=== Test Summary');
+    expect(outputPanelText).toContain('TEST NAME');
+    expect(outputPanelText).toContain('ended SFDX: Run Apex Tests');
+  });
+
+  step('Re-run Last Apex Test Class', async () => {
+    const workbench = await browser.getWorkbench();
+    const testingView = await workbench.getActivityBar().getViewControl('Testing');
+    let textEditor: TextEditor;
```

Comment:
I general, we prefer the declaration and assignment of variables to be on a single line, and to declare as `const` (unless there is a reason to do so).  Please remove line 157, and then change line 170 to:
`const textEditor = await editorView.openEditor...`